charts/kube2iam/_kube2iam/daemonset.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kube2iam
+  labels:
+    app: kube2iam
+spec:
+  template:
+    metadata:
+      labels:
+        name: kube2iam
+    spec:
+      hostNetwork: true
+      containers:
+        - image: jtblin/kube2iam:latest
+          name: kube2iam
+          args:
+            - "--base-role-arn=arn:aws:iam::593291632749:role/"
```

Comment:
So I understand well this will be concatenated by `kube2iam` with the pod's annotation `iam.amazonaws.com/role` which will contain the role name.

So we probably change this to `...role/users"` unless we want to pass `users/dev_kerin`, etc.. all the times which is error prone.